test/browsercontext.spec.js

```diff
@@ -389,7 +396,7 @@ module.exports.describe = function({testRunner, expect, playwright, CHROMIUM, FF
       const context = await browser.newContext();
       const page = await context.newPage();
       const [otherPage] = await Promise.all([
-        new Promise(r => context.once('page', async event => r(await event.page()))),
+        context.waitForEvent('page').then(async event => await event.page()),
```

Comment:
nit: arrow function can we written as `...then(e => e.page())`

Reply:
Done.